lib/CoreBot.js

```diff
@@ -1007,7 +1007,7 @@ function Botkit(configuration) {
         }
 
         if (typeof(events) == 'string') {
-            events = events.split(/\,/g);
+            events = events.replace(/ /g, '').split(/\,/g);
```

Comment:
This update will inhibit the use of multiple keyword events. For example, if at some point we want to use "super channel" instead of "super_channel", this would be a problem. I'm suggesting to use `.trim()` for every splitted keyword.

What do you think?